carma-messenger-core/Dockerfile

```diff
@@ -39,9 +39,7 @@ LABEL org.label-schema.vcs-url="https://github.com/usdot-fhwa-stol/carma-messeng
 LABEL org.label-schema.vcs-ref=${VCS_REF}
 LABEL org.label-schema.build-date=${BUILD_DATE}
 
-COPY --from=setup /opt/carma/install /opt/carma/install
 COPY --from=setup /opt/carma/install_ros2 /opt/carma/install_ros2
```

Comment:
I think now because this repo only has ROS2, I think it makes sense not to distinguish as install_ros2. Could you rename all those instances of " * _ros2"?